lib/coll.gi

```diff
@@ -3052,6 +3053,31 @@ end);
 InstallMethod( CanComputeIsSubset,"default: no, unless identical",
   [IsObject,IsObject],IsIdenticalObj);
 
+# avoid immediate methods triggering multiple type changes once the Size of
+# an object is known.
```

Comment:
This comment starts mid-thought. Perhaps: "Custom SetSize method which sets various properties implied by the given size. This avoids ..." ?

Reply:
I did not see this as mid-thought, but I'm happy to change it.